app/Helpers/ScoutHelper.php

```diff
@@ -2,27 +2,84 @@
 
 namespace App\Helpers;
 
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Facades\DB;
+
 class ScoutHelper
 {
     /**
      * When updating a model, this method determines if we should update the search index.
      *
-     * @return bool
-     *
      * @codeCoverageIgnore
      */
-    public static function activated()
+    public static function activated(): bool
```

Comment:
Technically, if it's a boolean, the function should be `isActivated` or something like that.